apps/pie-docs/src/assets/styles/components/_showMore.scss

```diff
@@ -0,0 +1,22 @@
+@use '../fozzie-config' as f;
+
+.c-showMore-backdrop {
```

Comment:
To fit with the naming scheme, it would make more sense to call this class the `c-showMore` and then the button class below `c-showMore-btn`